server/rating_service/rating_service.py

```diff
@@ -141,6 +140,10 @@ async def _handle_rating_queue(self) -> None:
     async def _rate(self, summary: GameRatingSummary) -> None:
         assert self._rating_type_ids is not None
 
+        if summary.rating_type is None:
+            self._logger.debug(f"Not processing game {summary.game_id} since it is not rated.")
```

Comment:
We don't use f-strings in log messages.

```suggestion
            self._logger.debug("Not processing game %s since it is not rated.", summary.game_id)
```